shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -58,6 +59,11 @@
 // change. Unfortunately unless you have Werror turned on, incompatible pointers as arguments are
 // just a warning.
 @interface FlutterViewController () <FlutterBinaryMessenger, UIScrollViewDelegate>
+// TODO(dkwingsmt): Make the view ID property public once the iOS shell
+// supports multiple views.
+// https://github.com/flutter/flutter/issues/138168
+@property(nonatomic, assign) int64_t viewId;
```

Comment:
[The runner API design](https://docs.google.com/document/d/1WkJuzifgJXyceaSoCCu8qTETYmYBOYKoz7Qaxtsore4/edit?tab=t.0#heading=h.jony3helpbsu) had the following property, which was Stuart's recommendation:

```suggestion
@property(nonatomic, readonly) int64_t viewIdentifier;
```

Reply:
Did he mention anywhere the reason for this? Does it only apply to Objective-C code or also the embedder API?

Reply:
I think "fooIdentifier" is our preferred naming convention for Darwin platforms

I'm an Objective-C noob, so not sure about `readonly` versus `assign`, `assign` might be perfectly acceptable. I don't feel strongly about this one.